tests/unit/providers/test_provider.py

```diff
@@ -38,12 +38,7 @@ def provider_class(request):
         ),
     ],
 )
-def test_is_base_available(
-    provider_class,
-    name,
-    expected_valid,
-    expected_reason,
-):
+def test_is_base_available(provider_class, name, expected_valid, expected_reason):
```

Comment:
odd, why did this get re-formatted?

Reply:
I snuck that in on purpose.

Sometimes I add enough arguments that `black` breaks it into multiple lines to satisfy the line length requirement.  Later, I remove some arguments and forget to squash the function back into one line.